aif360/algorithms/postprocessing/deterministic_reranking.py

```diff
@@ -0,0 +1,212 @@
+import numpy as np
+import pandas as pd
+
+from aif360.algorithms import Transformer
+from aif360.datasets import StructuredDataset, RegressionDataset
+
+class DeterministicReranking(Transformer):
+    """A collection of algorithms for construction of fair ranked candidate lists. [1]_ .
+
+    References:
+        .. [1] KDD '19: Proceedings of the 25th ACM SIGKDD 
+           International Conference on Knowledge Discovery & Data Mining, July 2019, Pages 2221-2231.
+    """
+
+    def __init__(self,
+            unprivileged_groups: list[dict],
+            privileged_groups: list[dict]):
```

Comment:
This type hint doesn't work in Python 3.8. You can change it to use `List` from `typing`. Type hints are optional for this package, though. We just require the type in the docstring